corehq/apps/locations/migrations/0013_get_stock_levels_from_config.py

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+from south.v2 import DataMigration
+from dimagi.utils.couch.database import get_db
+from corehq.apps.domain.models import Domain
+from corehq.apps.locations.models import LocationType
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """call the save method on all relevant location types"""
+        db = get_db()
+        for domain in Domain.get_all():
+            if domain.commtrack_enabled:
```

Comment:
a bit confused by this. is the thinking that we only care about setting the stock thresholds if commtrack is enabled on the domain?

it seems like it might be easier to just save every `LocationType` in the DB regardless of domain but maybe there's a good reason not to do that.